internal/raft/replicator_test.go

```diff
@@ -44,6 +45,33 @@ func testListMembers(t *testing.T) {
 	clus.assertMembers(t, members)
 }
 
+func testSaveLoadLargeData(t *testing.T) {
+	var reqs []*kvReq
+	var iterations = 20
```

Comment:
Please replace this line with `iterations := 20`

Reply:
Done